crates/core/machine/src/syscall/precompiles/README.md

```diff
@@ -208,8 +208,7 @@ Then implement the tests and run `cargo prove build` to generate an ELF file.
 ### Include the ELF File in `program.rs`
 In your main SP1 project, include the generated ELF file by updating `program.rs`. 
```

Comment:
I think this step is no longer necessary

Reply:
Oh yeah it's `crates/test-artifacts/src/lib.rs` now